pages/manage/productions/index.vue

```diff
@@ -240,6 +240,10 @@
     a {
       font-weight: 400;
       color: $udb-blue;
+
+      &:hover {
+        cursor: pointer;
+      }
```

Comment:
This will probably be an issue for every `a` tag that doesn't have an `href` though.
So we should probably add it to some more global scope?